__tests__/users/App.test.js

```diff
@@ -49,6 +50,8 @@ describe('App Component', () => {
           body: JSON.stringify({
             ...filteredUsersData,
             users: filteredUsersData.users.filter((user) => user.discordId),
+            ...mockUserData,
+            users: mockUserData.users.filter((user) => user.discordId),
```

Comment:
Why are you filtering users by discord ID?

Reply:
filtering of users by Discord ID in the test is intentional to ensure that our application accurately handles and displays data specific to users with Discord IDs. This approach allows us to comprehensively test our application's functionality. If you have any further questions or suggestions, please let me know.